jOOQ/src/main/java/org/jooq/impl/DefaultBinding.java

```diff
@@ -2389,6 +2391,29 @@ private static final String preparse(String formatted, boolean includeDate) {
 
         // Probably a bug, let OffsetDateTime or OffsetTime report it
         else {
+
+            // [#7126] Postgresql can return Special Date/Time Inputs such as "epoch", "infinity", "now", ...
+            if ("epoch".equals(string)) {
+                return "1970-01-01T00:00Z";
+            } else if ("infinity".equals(string)) {
+                // DATE_POSITIVE_INFINITY taken from org.postgresql.PGStatement
+                return "+292278994-08-16T23:00Z";
+            } else if ("-infinity".equals(string)) {
+                // DATE_NEGATIVE_INFINITY taken from org.postgresql.PGStatement
+                return "-292275055-05-16T23:00Z";
+            } else if ("now".equals(string)) {
+                return OffsetDateTime.now().toString();
```

Comment:
Hmm, the unfortunate side-effect of this usage is that `now` is an expression on the PostgreSQL *server*, which is now evaluated on the *client* (which may be off by whatever number of seconds/minutes/timezones/etc)